pvnet_app/model_configs/all_models.yaml

```diff
@@ -70,9 +70,8 @@ models:
     ecmwf_only: True
     uses_satellite_data: False
     uses_ocf_data_sampler: False
-# The day ahead model has not yet been re-trained with data-sampler.
-# It will be run with the legacy dataloader using ocf_datapipes
-  - name: pvnet_day_ahead
+# Legacy day ahead model without data-sampler
+  - name: pvnet_day_ahead_legacy
```

Comment:
do you mind calling it `pvnet_day_ahead`. The name is important in the blend further down the line. I hope they can both be called the same thing